seisbench/models/deepdenoiser.py

```diff
@@ -284,3 +283,454 @@ def forward(self, x, mid):
+                        Default are 2 channels.
+    :param sampling_rate: Sampling rate of the waveform data in Hz.
+                          Default sampling rate is 100 Hz.
+    :param filters_root: Number of filters in the first convolutional layer (doubles with depth).
+                         Default is 8.
+    :param depth: Number of encoding/decoding levels in the U-Net.
+                  Default is 6 for STFT
+    :param kernel_size: Kernel size for convolutional layers.
+                        Default is (3, 3).
+    :param strides: Stride size used for down/upsampling using Conv2D and transpose Conv2D layers.
+                    Default is (2, 2).
+    :param output_activation: Activation function applied to final output
+                              Default is Softmax.
+    :param drop_rate: Dropout rate used throughout the network.
+                      Default drop_rate is 0
+    :param use_bias: Whether to use bias in convolutional layers.
+                     Default is False.
+    :param norm: Type of normalization applied to traces ("peak" or "std").
+                 Default is "peak"
+    :param nfft: Length of the FFT used, if a zero padded FFT is desired for scipy STFT.
+                 If None, the FFT length is nperseg.
+    :param nperseg: Length of each segment for scipy STFT. Default is 60
+    :param attention: Whether to use attention gates in skip connections or not.
+                      Default is False
+    :param kwargs: Additional arguments passed to the base `WaveformModel`.
+    """
+
+    _annotate_args = WaveformModel._annotate_args.copy()
+    _annotate_args["overlap"] = (_annotate_args["overlap"][0], 1500)
+    _annotate_args["blinding"] = (
+        "Number of prediction samples to discard on each side of each window prediction",
+        (0, 0),
+    )
+
+    def __init__(
+        self,
+        in_samples: int = 3000,
+        in_channels: int = 2,
+        sampling_rate: float = 100,
+        filters_root: int = 8,
+        depth: int = 6,
+        kernel_size: tuple[int, int] = (3, 3),
+        strides: tuple[int, int] = (2, 2),
+        output_activation=torch.nn.Softmax(dim=1),
+        drop_rate: float = 0.0,
+        use_bias: bool = False,
+        norm: str = "peak",
+        nfft: int = 60,
+        nperseg: int = 30,
+        attention: bool = False,
+        **kwargs,
+    ):
+
+        citation = (
+            "Heuel, J., & Friederich, W. (2022). "
+            "Suppression of wind turbine noise from seismological data using nonlinear thresholding "
+            "and denoising autoencoder. "
+            "Journal of Seismology, 26(5), 913-934. "
+            "https://doi.org/10.1007/s10950-022-10097-6"
+        )
+
+        WaveformModel.__init__(
+            self,
+            citation=citation,
+            in_samples=in_samples,
+            output_type="array",
+            default_args={"overlap": in_samples // 2},
```

Comment:
Setting this will make it impossible to pass `default_args` through the kwargs, so it should be removed. It is also not necessary, because it is set before (see comment above).